mojaloop/values.yaml

```diff
@@ -4981,88 +7031,239 @@ thirdparty:
       # - name: LOG_LEVEL
       #   value: debug
 
+    ## Svc configs
     config:
-      production.json: {
-        "PORT": 4004,
-        "HOST": "0.0.0.0",
-        "PARTICIPANT_ID": "centralauth",
-        "REDIS": {
-            "PORT": 6379,
-            "HOST": "auth-svc-redis-svc",
-            "TIMEOUT": 100
-        },
-        "INSPECT": {
-            "DEPTH": 4,
-            "SHOW_HIDDEN": false,
-            "COLOR": true
-        },
-        "SHARED": {
-            "THIRDPARTY_REQUESTS_ENDPOINT": "tp-api-svc:3008",
-            "ALS_ENDPOINT": "$release_name-account-lookup-service",
-            "JWS_SIGN": false,
-            "JWS_SIGNING_KEY": "/secrets/jwsSigningKey.key",
-            "WSO2_AUTH": {
-                "staticToken": "0706c62f-c022-3c42-8d14-8b5df128876d",
-                "tokenEndpoint": "",
-                "clientKey": "test-client-key",
-                "clientSecret": "test-client-secret",
-                "refreshSeconds": 3600
-            },
-            "TLS": {
-                "mutualTLS": {
-                    "enabled": false
-                },
-                "creds": {
-                    "ca": "/secrets/client.crt",
-                    "cert": "/secrets/client.crt",
-                    "key": "/secrets/client.crt"
-                }
-            }
-        },
-        "DATABASE": {
-            "client": "mysql",
-            "version": "5.5",
-            "connection": {
-                "host": "mysql-auth-svc",
-                "port": 3306,
-                "user": "auth-svc",
-                "password": "password",
-                "database": "auth-svc",
+      ## DB Configuration
+      db_host: *TP_AUTH_SVC_DB_HOST
+      db_port: *TP_AUTH_SVC_DB_PORT
+      db_user: *TP_AUTH_SVC_DB_USER
+      ## Secret-Management
+      ### Set this if you are using a clear password configured in the config section
+      db_password: *TP_AUTH_SVC_DB_PASSWORD
```

Comment:
```suggestion
      db_password: ''
```

Reply:
@elnyry-sam-k this is arelady set on the top-level config.